Vimeo-DotNet/VimeoAPI/packages.config

```diff
@@ -1,4 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Newtonsoft.Json" version="5.0.5" targetFramework="net40" />
+  <package id="Newtonsoft.Json" version="6.0.8" targetFramework="net46" />
```

Comment:
Yep - I had pulled the most recent version from TechSmith, not NuGet: fixed in 40f1a31.